Traceback (most recent call last):
  File "C:\Users\jiral\AppData\Local\Programs\Python\Python310\lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\jiral\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 1269, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\jiral\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\util.py", line 85, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\jiral\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\util.py", line 60, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\jiral\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 646, in run_until_complete
    return future.result()
  File "C:\Users\jiral\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 707, in async_execute
    await self.async_execute_cell(
  File "C:\Users\jiral\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 1025, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\jiral\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 919, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
cislo = int(input("Zadej cislo: "))
print(cislo**2)
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mStdinNotImplementedError[0m                  Traceback (most recent call last)
Cell [1;32mIn [26], line 1[0m
[1;32m----> 1[0m cislo [38;5;241m=[39m [38;5;28mint[39m([38;5;28minput[39m([38;5;124m"[39m[38;5;124mZadej cislo: [39m[38;5;124m"[39m))
[0;32m      2[0m [38;5;28mprint[39m(cislo[38;5;241m*[39m[38;5;241m*[39m[38;5;241m2[39m)

File [1;32m~\AppData\Local\Programs\Python\Python310\lib\site-packages\ipykernel\kernelbase.py:1174[0m, in [0;36mKernel.raw_input[1;34m(self, prompt)[0m
[0;32m   1167[0m [38;5;250m[39m[38;5;124;03m"""Forward raw_input to frontends[39;00m
[0;32m   1168[0m 
[0;32m   1169[0m [38;5;124;03mRaises[39;00m
[0;32m   1170[0m [38;5;124;03m------[39;00m
[0;32m   1171[0m [38;5;124;03mStdinNotImplementedError if active frontend doesn't support stdin.[39;00m
[0;32m   1172[0m [38;5;124;03m"""[39;00m
[0;32m   1173[0m [38;5;28;01mif[39;00m [38;5;129;01mnot[39;00m [38;5;28mself[39m[38;5;241m.[39m_allow_stdin:
[1;32m-> 1174[0m     [38;5;28;01mraise[39;00m StdinNotImplementedError(
[0;32m   1175[0m         [38;5;124m"[39m[38;5;124mraw_input was called, but this frontend does not support input requests.[39m[38;5;124m"[39m
[0;32m   1176[0m     )
[0;32m   1177[0m [38;5;28;01mreturn[39;00m [38;5;28mself[39m[38;5;241m.[39m_input_request(
[0;32m   1178[0m     [38;5;28mstr[39m(prompt),
[0;32m   1179[0m     [38;5;28mself[39m[38;5;241m.[39m_parent_ident[[38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m],
[0;32m   1180[0m     [38;5;28mself[39m[38;5;241m.[39mget_parent([38;5;124m"[39m[38;5;124mshell[39m[38;5;124m"[39m),
[0;32m   1181[0m     password[38;5;241m=[39m[38;5;28;01mFalse[39;00m,
[0;32m   1182[0m )

[1;31mStdinNotImplementedError[0m: raw_input was called, but this frontend does not support input requests.
StdinNotImplementedError: raw_input was called, but this frontend does not support input requests.

